README.md

```diff
@@ -126,7 +126,7 @@ They should should focus on the cli operation and are not a replacement for pulp
 i.e. make sure the cli translates to the right api calls, but do not care about pulp internals.
 In order to perform, a running instance of pulp with all necessary plugins installed must be configured in `tests/scripts/config/pulp/settings.toml`.
 All tests are excercised using `make test`.
-Individual tests can be selected by pattern via `pytest -k file`.
+Individual tests can be selected by pattern via `pytest -k pulp_file`.
```

Comment:
One thing I am trying to correct for is the possibility that tests outside of pulp_file might contain "file" which would match this expression.

Reply:
What if we look for "test_file" then you could still have "test_rpm_do_sth_with_file".

Reply:
It's still a problem though if there's core test that starts with "file_..." since the core tests aren't namespaced.